package.json

```
@@ -4,14 +4,19 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "npm-run-all build:*",
-    "build:scss": "node-sass 'assets/scss' -o 'dist/assets/css' --output-style compressed",
-    "dev": "nodemon -e js,scss --exec \"npm run lint\" --ignore dist",
+    "build": "npm-run-all build:clean build:html build:js",
+    "build:clean": "rm -rf dist/",
+    "build:html": "env-cmd ejs-prerender",
+    "build:js": "webpack --config webpack.prod.js --mode production",
+    "dev": "run-p watch:*",
```

Rather than `run-p` it would be easier to track the source of this command if you used the slightly more verbose flag syntax.
```suggestion
    "dev": "npm-run-all -p watch:*",
```